cristin-organization/src/test/java/no/unit/nva/cristin/organization/organization.feature

```diff
@@ -22,15 +22,15 @@ Feature: API tests for Cristin Organization retrieve and search
     Then status 400
     And match response.title == 'Bad Request'
     And match response.status == 400
-    And match response.detail == 'Invalid path parameter for identifier, needs to be a number....'
+    And match response.detail == 'Invalid path parameter for identifier, needs to be four numbers separated by dots'
```

Comment:
"organization identifier matching pattern /(?:\d+\.){3}\d+/, e.g. (100.0.0.0)"

Reply:
Reminds me of something I saw on Reddit where the error message sent back to the user was a regex. User friendliness

Reply:
Changed text